.pre-commit-config.yaml

```diff
@@ -12,7 +12,7 @@ repos:
     -   id: rust-style
         name: Rust style
         description: Run cargo fmt on files included in the commit. rustfmt should be installed before-hand.
-        entry: cargo +nightly fmt --all -- --check
+        entry: cargo fmt --all -- --check
```

Comment:
any reason to remove the +nightly?

Reply:
It doesn't work, I get
```
error: no such subcommand: `+nightly`
```
when trying to commit any Rust code.